### PR TITLE
Add module to create template help text

### LINKED ISSUE #32259: flutter create -t module doesn't appear in flutter create -h
```
Create a new Flutter project.

If run on a project that already exists, this will repair the project, recreating any files that are missing.

Usage: flutter create <output directory>
-h, --help                     Print this usage information.
    --[no-]pub                 Whether to run "flutter packages get" after the project has been created.
                               (defaults to on)

    --[no-]offline             When "flutter packages get" is run by the create command, this indicates whether to run it in offline mode or not. In offline mode, it will need to have all dependencies already available in the pub cache to succeed.
    --[no-]with-driver-test    Also add a flutter_driver dependency and generate a sample 'flutter drive' test.
-t, --template=<type>          Specify the type of project to create.

          [app]                (default) Generate a Flutter application.
          [package]            Generate a shareable Flutter project containing modular Dart code.
          [plugin]             Generate a shareable Flutter project containing an API in Dart code with a platform-specific implementation for Android, for iOS code, or for both.

-s, --sample=<id>              Specifies the Flutter code sample to use as the main.dart for an application. Implies --template=app. The value should be the sample ID of the desired sample from the API documentation website (http://docs.flutter.dev). An example can be found at
                               https://master-api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html

    --list-samples=<path>      Specifies a JSON output file for a listing of Flutter code samples that can created with --sample.
    --[no-]overwrite           When performing operations, overwrite existing files.
    --description              The description to use for your new Flutter project. This string ends up in the pubspec.yaml file.
                               (defaults to "A new Flutter project.")

    --org                      The organization responsible for your new Flutter project, in reverse domain name notation. This string is used in Java package names and as prefix in the iOS bundle identifier.
                               (defaults to "com.example")

    --project-name             The project name for this new Flutter project. This must be a valid dart package name.
-i, --ios-language             [objc (default), swift]
-a, --android-language         [java (default), kotlin]

Run "flutter help" to see global options.
```

### PR DESCRIPTION
**Pull request by jmagman**
## Description
Add module to help text.
```
$ flutter create -h
...
-t, --template=<type>          Specify the type of project to create.

          [app]                (default) Generate a Flutter application.
          [module]             Generate a Flutter project to add Flutter views to a non-Flutter Android or iOS application.
          [package]            Generate a shareable Flutter project containing modular Dart code.
          [plugin]             Generate a shareable Flutter project containing an API in Dart code with a platform-specific implementation for Android, for iOS code, or for both.
...
```

## Related Issues

Fixes https://github.com/flutter/flutter/issues/32259.

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/create.dart [hunk 1]
```diff
@@ -85,6 +85,8 @@ class CreateCommand extends FlutterCommand {
         getEnumName(_ProjectType.plugin): 'Generate a shareable Flutter project containing an API '
             'in Dart code with a platform-specific implementation for Android, for iOS code, or '
             'for both.',
+        getEnumName(_ProjectType.module): 'Generate a Flutter project to add Flutter views to a non-'
```
Comment (jmagman) [inline packages/flutter_tools/lib/src/commands/create.dart]: I couldn't find an official sentence for this, so I made it up.  Open to suggestions, @xster.

Comment (xster) [reply packages/flutter_tools/lib/src/commands/create.dart]: How about `Generate a project to add a Flutter module to existing Android or iOS applications`?

@RedBrogdon?

Comment (jmagman) [reply packages/flutter_tools/lib/src/commands/create.dart]: @zanderso

Comment (jmagman) [reply packages/flutter_tools/lib/src/commands/create.dart]: @zanderso

Comment (zanderso) [reply packages/flutter_tools/lib/src/commands/create.dart]: I like @xster's suggestion with a tiny change:
`Generate a project to add a Flutter module to an existing Android or iOS application.`

**Comment by xster** [comment]
LGTM

File: packages/flutter_tools/lib/src/commands/create.dart (hunk 1 above)
Comment (jmagman) [reply packages/flutter_tools/lib/src/commands/create.dart]: I like that, thanks!

**Pull request merged**